packages/cypress/package.json

```diff
@@ -59,6 +59,7 @@
   },
   "devDependencies": {
     "@chromatic-com/shared-e2e": "workspace:*",
+    "@rrweb/types": "^2.0.0-alpha.18",
```

Comment:
The latest rrweb has moved the types to a separate package, requiring this and all the import changes below.